packages/ckeditor5-image/src/imageupload/ui/imageuploadpanelview.js

```diff
@@ -213,6 +215,17 @@ export default class ImageUploadPanelView extends View {
 		}, { priority: 'high' } );
 	}
 
+	/**
+	 * Returns a view of the integration.
+	 *
+	 * @param {string} name The name of the integration.
+	 *
+	 * @returns {module:ui/view~View}
+	 */
+	getIntegration( name ) {
```

Comment:
Since you implemented this method, maybe let's use it across https://github.com/ckeditor/ckeditor5/blob/i/7878/packages/ckeditor5-image/tests/imageupload/ui/imageuploadpanelview.js whenever possible? (instead of `view._integrations.get()`).